integration/kubernetes/overhead/test_ntttcp/check_before_get_overhead.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+max_tries="180"
+try=1
+
+client="ntttcp-client"
+
+while ! kubectl logs "${client}" | grep "INFO: Network activity progressing" >/dev/null; do
```

Comment:
This check seems like it might be fragile - what if that message changes? Is there no message identifier to check or a command whose return code you can check instead?